clang/test/dpct/macro_test.cu

```diff
@@ -1153,6 +1153,10 @@ template<class T1, class T2, int N> __global__ void foo31();
 #define FOO31(DIMS) foo31<unsigned int, float, DIMS><<<1,1>>>();
 
 //CHECK:   q_ct1.submit([&](sycl::handler &cgh) {
+//CHECK-NEXT:     /*
+//CHECK-NEXT:     DPCT1099:{{[0-9]+}}: The size of the accessor is a literal value to avoid
+//CHECK-NEXT:     potential out of scope issue of the used macro/const value.
+//CHECK-NEXT:     */
 //CHECK-NEXT:     sycl::local_accessor<double, 2> red_acc_acc_ct1(
 //CHECK-NEXT:         sycl::range<2>(8 /*8*/, 8 /*BLOCK_PAIR / SIMD_SIZE*/), cgh);
```

Comment:
```suggestion
//CHECK-NEXT:         sycl::range<2>(8, 8 /*BLOCK_PAIR / SIMD_SIZE*/), cgh);
```
/*8*/ is not a macro, why do we need to add it in comment in addition to value?

Reply:
It is a bug about macro source location finding exactly. But it is it not related to this issue, and I prefer to fix it in a separated PR.